language/Types.md

```diff
@@ -74,7 +74,16 @@ instance showPercentage :: Show Percentage where
 
 ## Functions
 
-Functions in PureScript are like their Javascript counterparts, but always have exactly one argument.
+Functions in PureScript are like their Javascript counterparts, but always have exactly one argument. Function declarations and lambdas (anonymous functions) with multiple arguments are shorthand for curried lambda abstractions. For example, the following are all equivalent:
+
+```purescript
+add x y = x + y
+add = \x y -> x + y
+add = \x -> \y -> x + y
+add = (+)
+```
+
+Despite this overwhelming convention, uncurried JavaScript functions are can be used through the [purescript-functions](https://pursuit.purescript.org/packages/purescript-functions/) library. For more information see [The Foreign Function Interface](http://www.purescript.org/learn/ffi/).
```

Comment:
Ugh diction. Never works the first time (unlike PureScript!)

Is this better? (also changed the link to point to the relevant section)
> Since uncurried JavaScript functions can be more efficient to run, they are provided indirectly through the [purescript-functions](https://pursuit.purescript.org/packages/purescript-functions/) library. This is unnecessary, however, in most cases and should **not** be used for APIs. For more information see [Functions of Multiple Arguments](http://www.purescript.org/learn/ffi/#functions-of-multiple-arguments).